src/virtualenv/app_data/via_disk_folder.py

```diff
@@ -155,6 +157,14 @@ def write(self, content):
             folder.mkdir(parents=True, exist_ok=True)
         except OSError:
             pass
+        with tempfile.NamedTemporaryFile("w", prefix="atomic-write", delete=False) as tf:
```

Comment:
I'd prefer this functionality to be within a ``write_file_atomic`` function (maybe in ``utils``), than to complicate the body of this function.

Reply:
Going to move into util/path/_sync.py